src/main/java/com/agiletec/plugins/jacms/aps/system/services/content/model/ContentDto.java

```diff
@@ -279,20 +270,40 @@ private void fillAbstractResourceAttribute(AttributeInterface attribute, EntityA
         }
     }
 
+    private Map<String, String> getAddionalLinkAttributes (final EntityAttributeDto attributeDto) {
+        final Map<String, String> linkProperties = new HashMap<>();
+        final String rel = (String)((Map) attributeDto.getValue()).get("rel");
+        if (rel != null ) {
+            linkProperties.put("rel", rel);
+        }
+        final String target = (String)((Map) attributeDto.getValue()).get("target");
+        if (target != null ) {
+            linkProperties.put("target", target);
+        }
+        final String hreflang = (String)((Map) attributeDto.getValue()).get("hreflang");
+        if (hreflang != null ) {
+            linkProperties.put("hreflang", hreflang);
+        }
+        return linkProperties;
+    }
+
     private void fillLinkAttribute(AttributeInterface attribute, EntityAttributeDto attributeDto) {
         if (LinkAttribute.class.isAssignableFrom(attribute.getClass())) {
             LinkAttribute linkAttribute = (LinkAttribute)attribute;
             SymbolicLink link = new SymbolicLink();
+            Map<String, String> addionalLinkAttributes = new HashMap<>();
```

Comment:
There is a typo here, @fedelocci Could you please change the object name to  additionalLinkAttributes?
Thanks

Reply:
did it